canonical-kubernetes/addons/jfrog/README.md

```diff
@@ -0,0 +1,135 @@
+# JFrog Artifactory Enterprise Addon to Canonical-Kubernetes Spell
+
+Select this Addon to install Artifactory Enterprise on a Canonical Distribution of Kubernetes (CDK) Cluster in the cloud (currently AWS only) and region you have already selected. After completing this installation JFrog Artifactory will be installed on your Kubernetes cluster. Once you apply the Artifactory licenses you will be able to use Artifactory as a Docker registry, Helm repo, NPM repo and more.
+
+## Prerequisites:
+
+- AWS:- Amazon AWS Account setup with all privileges required to create a Kubernetes Cluster, create persistent storage, and run load balancers.
+
+- AWS Secret and Key available.
+
+- Installation machine (where you install and run conjure-up)
+  -  sudo privileges
+  -  OS is certified by JFrog(http://www.jfrog.com) and Canonical(http://wwww.canonical.com)(e.g Ubuntu 16.04 LTS).
+  -  Conjure-up dependencies installed (python, jq, perl)
+
+
+- conjure-up installed. Please see http://conjure-up.io for more details on conjure-up. Please note that JFrog has tested against the self hosted controller option. Juju as A Service (Jaas) option has not been cerfified by JFrog.
+
+- Licenses for Artifactory Enterprise Edition 5.9 or later versions from JFrog. You can obtain Enterprise Edition trial licenses from https://jfrog.com/artifactory/free-trial/#enterprise
+
+## Basic installation
+
+1. Run "conjure-up canonical-kubernetes --bundle-add channels.yaml" where channels.yaml has the following text:
```

Comment:
Apparently, the docs are missing reference to it, but spell / add-on steps can actually include fragments and apply them automatically, which would make these instructions much cleaner.  An example of how to do this can be found at https://github.com/conjure-up/spells/blob/master/canonical-kubernetes/steps/01_select-network/after-input

If there are no objections, I will create a follow-up PR to move this into the add-on and cut another release once that is accepted.

Reply:
For reference, I created https://github.com/conjure-up/conjure-up/issues/1424 to fix the docs.